processes/consumer/flush.go

```diff
@@ -90,7 +90,7 @@ func Flush(ctx context.Context, inMemDB *models.DatabaseData, dest destination.B
 
 			if err != nil {
 				tags["what"] = "merge_fail"
-				slog.With(logFields...).Warn(fmt.Sprintf("Failed to execute %s...not going to flush memory", action), slog.Any("err", err))
+				slog.Error(fmt.Sprintf("Failed to execute %s...not going to flush memory", action), append(logFields, slog.Any("err", err))...)
```

Comment:
Does append work?

Reply:
Also, any reason not to just change `.Warn` to `.Error`? 

```go
slog.With(logFields...).Error(fmt.Sprintf("Failed to execute %s...not going to flush memory", action), slog.Any("err", err))
```

Reply:
It should https://go.dev/play/p/yeI7nHhZ9X8?v=goprev

Reply:
I can just do the simple level change for now.

Reply:
Good to know